stories/molecules/treemap-prototype/contributor-node.tsx

```diff
@@ -54,13 +52,16 @@ const NonMemoizedContributorNode = <Datum extends { id: string; value?: number;
           }}
         >
           <div className="grid gap-x-2" style={{ gridTemplateColumns: "auto 1fr", gridTemplateRows: "auto auto" }}>
-            <img
-              className="col-start-1 col-span-1 row-span-2"
-              src={avatarURL}
-              width="42"
-              height="42"
-              style={{ display: "block", borderRadius: "50%", border: "solid 2px white", flexShrink: 0, flexGrow: 0 }}
-            />
+            <picture>
```

Comment:
Refactor (blocking): No need for the [picture element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) at the moment as we only have one source for an image.

Reply:
Added that to fix nextjs native `img` tag warning https://nextjs.org/docs/messages/no-img-element

Reply:
In their example, they use it with source and image tags. In our case we only have one image so it doesn't make sense even though it gives a warning. The img tag could become a Next Image instead.